qa/L0_perf_deeprecommender/test.sh

```diff
@@ -48,9 +48,9 @@ STATIC_BATCH=1
 DYNAMIC_BATCH=1
 INSTANCE_CNT=1
 
-# Copy TF Model
-rm -fr tfmodels && mkdir -p tfmodels && \
-    cp -r $REPODIR/perf_model_store/deeprecommender_graphdef tfmodels/.
+# Copy TF, ONNX and PyTorch Models
```

Comment:
Even though the comment says to copy the onnx and pytorch models. I don't see that happening,

Reply:
Fixed